bundles/framework/layerlist/model/Scale.js

```diff
@@ -1,3 +1,5 @@
+import * as d3 from 'd3';
```

Comment:
We should just remove d3 from layerlist. It's only used to calculate throttling (for search) based on how long the query is.

Reply:
Then again it's used on VectorLayerPlugin as well to calculate some colors. If we need to have it always present, we can use it here as well. I'll take a closer look at this.